script.js

```diff
@@ -119,48 +281,11 @@ function createSearchArea() {
     options.setAttribute("value", episode.name);
     dropdownSelection.appendChild(options);
   });
-
-  //SEARCH INPUT
-  searchInput.classList.add("search-input");
-  searchInput.type = "text";
-  searchInput.setAttribute("placeholder", "Search");
-  searchInput.setAttribute("Name", "searchBar");
-  searchInput.setAttribute("label", "search-episodes");
-  searchBarWrapper.appendChild(searchInput);
-
-  /***** EPISODE COUNT WRAPPER *****/
-  counterWrapper.classList.add("counter-wrapper");
-  counterWrapper.appendChild(countH2);
-  searchBarWrapper.appendChild(counterWrapper);
-  countH2.classList.add("counter-heading");
 }
 
-/***** TV-MAZE LINK *****/
-
-function createTvMazeLink() {
-  const tvMazeWrapper = document.createElement("div");
-  tvMazeWrapper.classList.add("tv-maze-wrapper");
-  body.appendChild(tvMazeWrapper);
-  const tvMazeLink = document.createElement("a");
-  tvMazeLink.setAttribute("target", "_blank");
-  tvMazeLink.setAttribute("rel", "noreferrer noopener");
-  tvMazeLink.classList.add("tv-maze-link");
-  tvMazeLink.innerHTML = "SOURCED FROM TV-MAZE.COM";
-  tvMazeLink.href = "https://www.tvmaze.com";
-  tvMazeWrapper.appendChild(tvMazeLink);
-}
-
-/***** TILE AREA WRAPPER *****/
-
-function createCardsWrapper() {
-  body.appendChild(cardsWrapper);
-  cardsWrapper.classList.add("cards-wrapper");
-}
-
-/***** DISPLAY EPISODE CARDS *****/
-
+/***** CREATE EPISODE CARDS *****/
 function displayEpisodeCards(episodeList) {
-  cardsWrapper.innerHTML = ""; //why is this needed when the wrapper is empty of content anyway?
+  cardsWrapper.innerHTML = "";
```

Comment:
You will need to empty the content of the div that contains the list with the cards at the very beginning of the function because when dropdownSelection eventListener executes multiple times this function will add new episodes to the existing list of episodes each time(see the .appendChild method used in this function). The result will be a long list with the episodes multiplied. If you make sure you will empty the content of the cardsWrapper div at the beginning, it will display each time only the data you want to be displayed, it will not keep all the previous displayed episodes.